vllm/worker/hpu_model_runner.py

```diff
@@ -1226,7 +1241,7 @@ def trim_attn_metadata(self, metadata: AttentionMetadata) -> object:
         attention_metadata = subtuple(metadata, 'TrimmedAttentionMetadata', [
             'attn_bias', 'seq_lens_tensor', 'block_list', 'block_mapping',
             'block_usage', 'slot_mapping', 'is_prompt', 'block_indices',
-            'block_offsets'
+            'block_offsets', 'block_scales'
         ])
         return attention_metadata
```

Comment:
cpu test returned issue in line 301 of this file:

RuntimeError: Class values must be non-negative.

Reply:
Yeah, it turns out cpu doesn't allow out-of-bounds class values. I've added a fix for it in: https://github.com/HabanaAI/vllm-fork/pull/378/commits/a05fffca2eb53e8c6d2fe1deed45d73160195771